libs/datasets/timeseries.py

```diff
@@ -782,6 +782,39 @@ def add_new_cases(timeseries: MultiRegionTimeseriesDataset) -> MultiRegionTimese
     return new_timeseries
 
 
+def _calculate_modified_zscore(series: pd.Series, window: int = 10) -> pd.Series:
+    """Calculates zscore removing zeros"""
+    series = series.copy()
+    series[series == 0] = None
+    rolling_series = series.rolling(window=window, min_periods=3)
+    # Shifting one to exclude current datapoint
+    mean = rolling_series.mean().shift(1)
```

Comment:
Not clear why we shift these forward one day. If we don't want to calculate for today, why not drop the last value in the column?

Reply:
ahh had to think about it a bit but it's because we're running the calculation for every day.  so all of the means/std have to be shifted for the z-score calculation